attractors_pick: [-0.684,  0.099,  0.599,
                  -0.750,  0.186, 0.560,
                  -0.614, -0.013, 0.619]
attractors_place: [-0.592, 0.004, 0.345,
                   -0.700, 0.083, 0.340,                   
                   -0.724, 0.180, 0.320]